ci-operator/step-registry/ipi/conf/aws/kms-key/ipi-conf-aws-kms-key-ref.yaml

```diff
@@ -18,12 +22,16 @@ ref:
     default: ""
     documentation: |-
       KMS Key for control plane nodes, if empty (""), read key id from "${SHARED_DIR}/aws_kms_key_arn" created by step aws-provision-kms-key
-  - name: AWS_KMS_KEY_ENABLE_COMPUTE
+  - name: AWS_KMS_KEY_ENABLE_DEFAULT_MACHINE
     default: "yes"
+    documentation: |-
+      Flag that indicate if provide KMS key in install-config.yaml for default machine nodes (platform.aws.defaultMachinePlatform.rootVolume.kmsKeyARN)
+  - name: AWS_KMS_KEY_ENABLE_COMPUTE
+    default: "no"
     documentation: |-
       Flag that indicate if provide KMS key in install-config.yaml for control plane nodes (controlPlane.platform.aws.rootVolume.kmsKeyARN)
```

Comment:
It seems the documentation of this var and the next var mixed up, `AWS_KMS_KEY_ENABLE_COMPUTE` -> `for control plane nodes`?

Reply:
fix by https://github.com/openshift/release/pull/37019